docs/Project-Configuration/Task-Runtime-Behavior.md

```diff
@@ -150,7 +150,8 @@ clean up the task directory.
 ## Task Timeouts
 
 Tasks are not allowed to run forever, so all commands that run for a task are
-subject to (configurable) timeouts. If a command hits a timeout, that command
+subject to (configurable) timeouts. However, tasks cannot be configured to have a timeout 
+greater than 86400 seconds (24 hours). If a command hits a timeout, that command
```

Comment:
This might belong better under [the section about exec timeout](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#timeout-handler).

Reply:
Looks like this comment got missed?

Reply:
Ah, I did see it and added it to the section you mentioned. 
But I didn't see the harm in having it mentioned in both sections